src/kdump-tools/Makefile

```diff
@@ -25,7 +25,11 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	stg import -s ../patch/series
 
 	# Build source and Debian packages
+ifeq ($(CROSS_BUILD_ENVIRON), y)
```

Comment:
To cross-compile kdump tools from source, a flag TARGET has to be passed to the makefile. Check README here: https://github.com/makedumpfile/makedumpfile/tree/0820a55bf9a0d1f6769398b686a328e5979542b5

But that isn't being set current, and i'm seeing compilation failures while trying for arm64. Has this been seen while testing for armhf?

to fix this, i had to edit the `override_dh_auto_build` inside the `src/kdump-tools/makedumpfile-1.6.1/debian/rules` to include TARGET=<CROSS_TARGET> flag and the compilation went through. 

Can you check and this add this flag?

Reply:
No, in armhf kdump tools builds fine without errors, just double checked it now. Trying to figure out what went wrong in arm64.

Reply:
OK, I added TARGET flag and it fixed arm64 build. Thanks for the help.